keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -1080,7 +1077,7 @@ N/A.
 
```

Comment:
Do we have some new findings for the `Troubleshooting` section?

Reply:
I'm not aware of that. We have worked with some users setting up MultiKueue, but it doesn't seem like the spec.managedBy would be bringing any particular reason to troublehooting (so far at least, and up to my knowledge).